packages/mdc-select/component.ts

```diff
@@ -156,6 +156,8 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> implements MDCR
     this.handleKeydown_ = (evt) => this.foundation_.handleKeydown(evt);
     this.handleMenuSelected_ = (evtData) => this.selectedIndex = evtData.detail.index;
     this.handleMenuOpened_ = () => {
+      this.foundation_.handleMenuOpened();
```

Comment:
This logic should be within the menu foundation. This seems like business logic that the select should not be aware of. Same with below.

Reply:
Did you mean the focus logic in `handleMenuOpened_` should be moved to select foundation? I feel it is out of the scope of this PR. There are many places in this file we do include business logic in component which should've been delegated to foundation.

Reply:
Ignore comment as it is out of scope for the PR